backend/tests/test-config.sh

```
@@ -43,7 +43,7 @@ get_admin_token() {
     # Use JWT admin endpoint
     local response=$(curl -s -X POST "$TEST_API_BASE/auth/admin/sessions" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)